README.md

```diff
@@ -2,8 +2,6 @@
 
 `s3x` is an open-source fork of `github.com/minio/minio` modified to work with TemporalX and an S3 gateway. It allows using IPFS from any application that currently uses S3, or minio, without needing to redesign your application architecture. It represents an incredible leap forward in usability for IPFS, and up until now no implementation of IPFS allowed you to use it, without needing to build your application specifically for IPFS. Additionally this means your S3 applications can leverage the benefits of IPFS as if it were a native IPFS application. One might say its better than...
 
-The long-term goal is to merge the TemporalX gateway upstream, but that will obviously depend on whether or not minio would accept it, although tif they were it is definitely something we will do.
```

Comment:
Are we removing this goal?

Reply:
No, I talked with one of the lead MinIO devs and they agreed once we're ready to port upstream we can, I removed because of this, but I guess I should reword instead